From dc5faa994b3103d7740e88c5de79969f6e58a237 Mon Sep 17 00:00:00 2001
From: Remi Collet <fedora@famillecollet.com>
Date: Thu, 24 Mar 2016 19:21:05 +0100
Subject: [PATCH] Fix compat with ZF 2.5

2.4: Zend\Cache\Exception\ExtensionNotLoadedException
2.5: Zend\ServiceManager\Exception\ServiceNotCreatedException

(cherry picked from commit 9474d5743122d4b1fc3bd5965d0eb798a93ad668)
---
 inc/session.class.php | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/inc/session.class.php b/inc/session.class.php
index f5deef5..dd572d7 100644
--- a/inc/session.class.php
+++ b/inc/session.class.php
@@ -578,7 +578,7 @@ static function loadLanguage($forcelang='') {
       try {
          $cache = Zend\Cache\StorageFactory::factory(array('adapter' => 'apc'));
          $TRANSLATE->setCache($cache);
-      } catch (Zend\Cache\Exception\ExtensionNotLoadedException $e) {
+      } catch (Exception $e) {
          // ignore when APC not available
          // toolbox::logDebug($e->getMessage());
       }
